woocommerce/woocommerce-gateway-stripe
# [ECE] fix fatal error "Uncaught Error: Call to a member function get_total() on null"

Fixes #3598 

## Testing instructions
- Enable the ECE feature flag.
- In the Stripe settings page, enable the express checkout methods.
- Go to "Pages > Add New" page.
- In `develop` branch, notice that a fatal error occurs.
- In this branch, there should be no fatal error. You should be able to create a new page.
